3.0.0/README.md

```diff
@@ -86,6 +87,20 @@ A text description of the tileset. The description can contain any valid unicode
 }
 ```
 
+## 3.6 `fillzoom`
+
+OPTIONAL. Default: null.
+
+An integer specifying the zoom level at which to generate overzoomed tiles from. Implementations may generate overzoomed tiles from parent tiles if the requested zoom level does not exist. In most cases, overzoomed tiles are generated from the maximum zoom level of the tileset. If fillzoom is specified, the overzoomed tile is generated from the fillzoom level.
+
+For example, in a tileset with maxzoom 10 and _no_ fillzoom specified, if a request for a z11 tile comes through, the implementation may use the maximum z10 parent tiles to generate the new, overzoomed z11 tile. If the same tilejson had fillzoom specified at z7, a request for a z11 tile could still try to generate with z10, but MAY backfill using to the z7 parent tile.
```

Comment:
`may use the maximum z10` -> `will use the maximum z10` seems like more correct wording, unless you meant `will try to use maximum z10 tile if it exists`

I would say:

```diff
-If the same tilejson had fillzoom specified at z7, a request for a z11 tile could still try to generate with z10, but MAY backfill using to the z7 parent tile.
+If the same tilejson had fillzoom specified at z7, a request for a z11 tile would use the z7 tile instead of z10.
```

I think the usecase here is for custom tile generation pipelines that want to have sparse tiling (avoid rendering to the maxzoom) for some regions, like large ocean regions.

Reply:
Awesome - thanks @springmeyer! https://github.com/mapbox/tilejson-spec/pull/50/commits/fcf4ebb703bb84f7bc287e1c61e743025cad4aa5